custom/aliases/example.aliases.sh

```diff
@@ -1,2 +1,9 @@
-# Add your own custom alias in the custom/aliases directory. Aliases placed
-# here will override ones with the same name in the main alias directory.
+# Add your own custom alias in the custom/aliases directory. Aliases placed here
+# will override ones with the same name in the main alias directory.
+# 
+# please note that you have to:
```

Comment:
In general, technical documentation tends to avoid using terms like "please" because when does one know when or not to use them?

I'd recommend having this be a title...

``` suggestion
# Usage
```